server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalAutoDateHistogramTests.java

```diff
@@ -108,10 +108,7 @@ public void testGetAppropriateRoundingUsesCorrectIntervals() {
         assertThat(result, equalTo(2));
     }
 
-
-    @Override
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/39497")
-    // TODO: When resolving the above AwaitsFix, just delete this override. Method is only overriden to apply the annotation.
```

Comment:
I've unmuted this test because the bug  _could_ have caused some of the failures we've seen previously, which seem to be off by one errors, such as in #39497. Some of those failures look quite similar:

```
   > Throwable #1: java.lang.AssertionError: expected:<{1548979200000=989}> but was:<{1548979200000=928, 1551398400000=61}>
```

Reply:
++ Agreed